src/Fixer/Strict/DeclareStrictTypesFixer.php

```diff
@@ -82,62 +82,48 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
             return;
         }
 
-        $sequence = $this->getDeclareStrictTypeSequence();
-        $sequenceLocation = $tokens->findSequence($sequence, $searchIndex, null, false);
+        $sequenceLocation = $tokens->findSequence([[T_DECLARE, 'declare'], '(', [T_STRING, 'strict_types'], '=', [T_LNUMBER], ')'], $searchIndex, null, false);
         if (null === $sequenceLocation) {
             $this->insertSequence($tokens); // declaration not found, insert one
 
             return;
         }
 
-        $this->fixStrictTypesCasing($tokens, $sequenceLocation);
-    }
-
-    /**
-     * @return Token[]
-     */
-    private function getDeclareStrictTypeSequence()
```

Comment:
Why is this removed. The new code is slower and harder to read now, since the line is so long?

Reply:
@GrahamCampbell this is removed because the new code is faster (yes, it is) and easier to read now - I'd say one 152 characters long line is more readable than 27 lines long function.

Reply:
I agree with @kubawerlos. Also static variables inside functions are easy memory leaks.

Reply:
To clarify:
- it is faster in runs that are not changing the code - for `findSequence` 6 new objects were created every time, and now none are created
- `getDeclareStrictTypeSequence` had to be updated anyway, as the check for `declare` could not use objects, because we have to search for any number, not only `1` - which is the core of this bugfix